src/containers/NFT/NFTTabs/test/NFTTabs.test.js

```diff
@@ -30,19 +31,29 @@ describe('NFT Transactions tab container', () => {
   it('renders table content', () => {
     const wrapper = createWrapper()
     expect(wrapper.find('.tabs').length).toBe(1)
-    expect(wrapper.find('a.tab').length).toBe(3)
-    expect(wrapper.find('a.tab').at(0).props().title).toBe('transactions')
-    expect(wrapper.find('a.tab').at(1).props().title).toBe('buy_offers')
-    expect(wrapper.find('a.tab').at(2).props().title).toBe('sell_offers')
-    expect(wrapper.find('a.tab.selected').text()).toEqual('transactions')
+    expect(wrapper.find('a.tab').length).toBe(2)
+    expect(wrapper.find('a.tab').at(0).props().title).toBe('buy_offers')
+    expect(wrapper.find('a.tab').at(1).props().title).toBe('sell_offers')
+    expect(wrapper.find('a.tab.selected').text()).toEqual('buy_offers')
     wrapper.unmount()
   })
 
-  it('renders transactions tab', () => {
-    const wrapper = createWrapper('transactions')
-    expect(wrapper.find('a.tab.selected').text()).toEqual('transactions')
-    wrapper.unmount()
-  })
+  // it('renders table content', () => {
+  //   const wrapper = createWrapper()
+  //   expect(wrapper.find('.tabs').length).toBe(1)
+  //   expect(wrapper.find('a.tab').length).toBe(3)
+  //   expect(wrapper.find('a.tab').at(0).props().title).toBe('transactions')
+  //   expect(wrapper.find('a.tab').at(1).props().title).toBe('buy_offers')
+  //   expect(wrapper.find('a.tab').at(2).props().title).toBe('sell_offers')
+  //   expect(wrapper.find('a.tab.selected').text()).toEqual('transactions')
+  //   wrapper.unmount()
+  // })
+
+  // it('renders transactions tab', () => {
+  //   const wrapper = createWrapper('transactions')
+  //   expect(wrapper.find('a.tab.selected').text()).toEqual('transactions')
+  //   wrapper.unmount()
+  // })
```

Comment:
shoudl be removed

Reply:
We will be adding all of this right back when nft_history is available.

Reply:
Yes, all the new changes are just temporary since nft_history is unavailable, and i thought I should just comment it out since Ill be the one who would be changing it back anyways. But I can still delete it if we want the files to look cleaner for a short while

Reply:
i get that, but you can have a branch for this. it's not good practice given that we use version control software

Reply:
no big deal just delete it